README.md

```diff
@@ -72,6 +72,7 @@ Check the return for a ```success``` flag. If success is set to true, then the u
 | maxRedirects         | Max number of redirects ogs will follow.                                   | 10            |          |
 | retry                | Number of times ogs will retry the request.                                | 2             |          |
 | encoding             | Setting this to `null` might help with running ogs on non english websites | utf8          |          |
+| headers              | Header for the request, is a object and be used for setting the user-agent | {}            |          |
```

Comment:
Consider: 'An object containing headers for the request. Useful for setting the user-agent`